modules/restore-replace-stopped-etcd-member.adoc

```diff
@@ -190,7 +190,14 @@ $ oc delete secret -n openshift-etcd etcd-serving-ip-10-0-131-183.ec2.internal
 $ oc delete secret -n openshift-etcd etcd-serving-metrics-ip-10-0-131-183.ec2.internal
 ----
 
-. Delete and re-create the control plane machine. After this machine is re-created, a new revision is forced and etcd scales up automatically. For more information, see "Replacing an unhealthy etcd member whose machine is not running or whose node is not ready".
+. Check whether a control plane machine set exists by entering the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-machine-api get controlplanemachineset output
```

Comment:
word: 'output' should not be part of command.

Reply:
Good catch - thanks! Should be fixed now.